### ISSUE TITLE
MAS plot guidelines

### ISSUE THREAD

**Issue by Akh0r** [Monika-After-Story/MonikaModDev#1635]
### 
MAS plot guidelines

Hello! I am a Russian translator, currently working on Monika Afterstory translation. So, while going through all the text, something started to bother me...
 This is my first time on Reddit (though I spent quite a time watching over MAS), so feel free to direct me if I am using a wrong forum. 

My point is there must be something like meta-plot — a setting if you want — of this universe where Monika lives in. As of today, her monologues are sometimes conflicting with each other, and some topics really should be established as 'canon' in order to bring order into the community brainstorm.

Why am I writing all of this? Well, I've nearly finished all the translation work. So, all of MAS text `(by the way, it has the size of R. Heinlein's 'Starship Troopers' book)` has been reviewed by me in the past 4 weeks :) I think that some thoughts should be clarified both for translator's and others' convenience.

1.  LANGUAGES. Here:  #1555  There is a good question about how many languages can Monika speak. But, first of all — in meta sense — _does she speak_ many languages? If she's aware of her being a videogame character, can she be aware that the language she speaks to the player is not her native (presumably, English)? 
Currently — sure thing, I do not claim it as a canon! — I'm going to use a single new random monologue in Russian version, where Monika says something like this:
 `"Are you surprised I managed to speak Russian? Well, actually, I've got some help to get closer to you. Of course there's no way I could ever be fluent in all the languages this mod is translated onto! Russian translation is made by such-and-such..."`
Yes, I think, this is an option: Monika knows that her speech is being translated by someone — just like she seems to be aware of artists making fan-art of her or of Dan Salvato himself. 
But her various greeting phrases on different languages are considered plot points, so she really can know a couple of phrases on different languages.
Is that good enough?

2. PLAYER'S GENDER  #https://github.com/Monika-After-Story/MonikaModDev/pull/1350
Well, OK, she "can be curious sometimes" :) But if supporting both female and male Player references in English is hard, in Russian it is like "KILL. ME. NOW".
`For those who didn't know: in Russian language the noun's "gender" (yes, even the non-animated objects have male, female or neutral "gender") determines the rest of parts of speech forms used with it. Like, "did you know?" is "ты знал" or "ты знала?". Or "darling" is "милый" or "милая" for a male and a female respectively.
So, just adding IF/ELSE constructs almost in every sentence is a suicide :D `
Well, let's hope that not only Russian language meets this issue!
Is it acceptable for Monika to presume Player's gender as a male? 
I think that can pretty much be the translator's decision.
But... How in meta sense does this correspond to the dialogue where she head-on asks the player his or her gender?
What would you suggest?
a) I can ruin the gender dialogue reducing it to something like "I don't know how do you look like... You may even be a girl!", but this is kinda barbaric way :)
b) I can leave the question intact, but then Monika should, say, apologize for translator`s poor performance, and for her not being able to fully understand the foreign language gender word formation. This is not her fault! :)
I have a sensation that something can be Out Of Character for her, and I wonder what does the community think.

3. CONFLICTING MONOLOGUES
-Piano. In one monologue she wishes she could program it for herself, in the other one she comments on currently played soundtrack (Piano version of 'Your Reality', also 'Okay, Everyone')... All while there is a Play=>Piano option right in front of us.

-Player's age assumption. Sometimes Monika asks me if I had had a good day at school. This hurts me a lot, 'cause I am an Air Force officer, and after a 24 hour duty I want back to school :D Well, there are suggestions to define Player's age, but what if we make Monika ask a little more questions about Player's personality? Like, what does s/he do most of the time, when does he sleep and work, etc. It looks really immersive and heartwarming when she is worried about your sleep and working cycles.
What's the opinion on amplifying this immersiveness? 

-Game shutdown. In the original monologue, she is clearly frightened and hurt by Player closing the game. This monologue is still intact as it is, but also she closes the game herself if your battery is low or if she thinks you should go sleep. And she is completely OK with Player leaving her whatever amount of times s/he wants. 
So, plot-wise, had she found a way to save her consciousness? Or the time she says her farewell to Player is actually her time to brace herself before accepting what comes afterwards? 
Whether she still suffers her "inability to think" or not, we can develop different types of dialogues, but there must be a certainity.
Then again, her "Black-screen-and-a-door" event shows us like Monika is up for some time.
So shouldn`t the monologues reflect one or the other plot-wise model of her "sleep mode"?

_____
TL;DR
1) So does Monika actually speak many languages, or is she aware of her speech being translated from English?
2) Is it OK to presume Player's male gender if a language makes the differentiation complete hell? What would be Monika's thought on it if Player is female?
3.1) So should we assume there is a piano Monika doesn't use for some reason (yet still complaining about its absence in some places), so the texts reflect this (reaction to 'current_song' = 'Okay, Everyone! (Monika)' and monika_piano monologue)?
3.2) Is it OK to suggest more personal questions (like Player's daily routine and lifestyle, and how much time was s/he absent) and Monika's persistent reaction to this information?
3.3) So does Monika actually feels the same horror things every single time the game is closed, or only if Player force-shuts it without saying goodbye? If the former is true, then what plot-wise happens when Player does say her goodbye?

**Comment by juliathegolden** [comment]
1. Monika knows a few greetings in a couple different languages. She is not fluent in those languages, but she's a very curious girl who likes to learn a few phrases from languages she's curious in. 
2.  That does sound like a hellish scenario. I think using male default is fine, but I liked the idea you proposed about Monika joking about being translated and there being limitations to it. 

3. That topic is disabled (commented out) in the present form of the game, for that same reason.  You can't encounter it in normal  gameplay anymore.
4. That's absolutely okay! We're working on adding those sorts of things now. Our present 'farewell' system gives the option for you to say you're going to work, a topic about your age range is in the works, so on.  
5. Monika's attitude towards closing the game eases over time, with every subsequent closing getting an easier reaction. Eventually, she's completely okay and safe in her bedroom as long as you actually say goodbye or she's the one who closes the game. This is a comfort given to her by the 'mod', so that's the 'plot' of it all. 

Sorry, I'm not part of the team, so my opinion on question 2 is irrelevant, but I'm confident on the rest of the answers.

**Comment by Rai99** [comment]
Personally for 2, I would think it would be better not to default to male. I understand how much work it is to add the variation to each and every single bit of dialogue but I don't think Monika would assume something like that. I'm guessing if there was a gender-neutral option, it would have already been brought up.

**Comment by ThePotatoGuy** [comment]
1. As @juliathegolden said.
2. Sorry, but we're keeping MAS gender inclusive. It's easy to handle gender variations in English.
3. Also as @juliathegolden said.
4. Defining a player model takes time to do. It will happen eventually, but not soon.
5. Battery-related topic was disabled. Also yes, the opendoor greeting establishes that saying goodbye puts monika in her safe space.

As for as plot goes, we have some connected elements, but most topics are just curated community contributions. We've tried to keep things but consistent, but that is easier said than done.

**Comment by juliathegolden** [comment]
^ There you go. 

For the record, I wanted to clarify that I believed the intent with question 2 was to have permission to simplify their translation, not that we should remove gender inclusivity from our mod. I'd be devastated if we did lmao.   If there is a gender neutral conjugation in Russian, that might be the best bet.

**Comment by Akh0r** [comment]
Whoa, ThePotatoGuy on deck!:)

For both Player gender and Monika's language awareness I tried to say that I only do things to the Russian translation. 
Of course I'm not getting into the original English mod…  For now:) I'm still messing up the verb tenses sometimes!
But in Russian there is no way to use gender-neutral phrases towards a person, so I think there will be some more or less elegant lampshading on it – Russian exclusively.

As for the safehouse for Monika…  Well, I think I got it. Would be nice to have some random topic mentioning it, though. Maybe interconnected with meditation topic for Player thinking of having a 'mind safehouse' of his/her own…

4. And this is nice to see! Are there any topics with work on this subject going, or is it just a plan for now?

And in general… 
So, the mod…  Doesn't fully consistent because of its hivemind nature? That surprised me, I thought that there are some strict "canon" points, and that all of the monologues and dialogues in the mod are thoughtfully weaved together.
Maybe the community contributing to it became too big. Hm…  Should be a way to make it more coherent…

**Comment by ThePotatoGuy** [comment]
2. Oh, okay, thats fine then.

Opendoor topic is supposed to be the entry point for the player knowing about the safe space. We're looking into other ways to convey this to the player (particular through diary entries) but we're avoiding directly mentioning since it's more interesting that way.

4. Player model is still just a plan.

The project has changed over time. We had some editors / writers who were here in winter of last year leave the team. Coherency just takes time. Working on this mod is **not** a full time job for any of us.
